cmd/server.go

```diff
@@ -54,6 +57,8 @@ func createHTTPServer(ctx api.Context) *echo.Echo {
 	e := echo.New()
 	e.HideBanner = true
 
+	e.Use(otelecho.Middleware("mission-control"))
```

Comment:
I think we should probably gate this and the db middleware if teacing is enabled only

Reply:
It works silently by using noop spans